pandas/_libs/intervaltree.pxi.in

```diff
@@ -69,7 +69,7 @@ cdef class IntervalTree(IntervalMixin):
         inclusive, closed = _warning_interval(inclusive, closed)
 
         if inclusive is None:
-            inclusive = "both"
+            inclusive = "right"
```

Comment:
is IntervalTree public? (or is the class the return type of a public function/method?)

maybe we don't need the deprecation here or don't even need to change at all.

is your understanding that the change from closed -> inclusive is for the public api or across the whole codebase?

Reply:
As far as I can see IntervalTree is private (There are no docs around). It obviously makes sense to rename the kwarg across the whole code base. But since its private, we don't have to be backwards compatible.

Reply:
yep. probably ok to remove the deprecation here as a follow-up